ui/pages/Project/components/ProjectOverview.jsx

```diff
@@ -282,10 +285,26 @@ const Dataset = React.memo(({ project, samplesByType, user }) => {
     })
   }
 
+  Object.assign(params, { namespace: project.workspaceNamespace, name: project.workspaceName })
+
+  const addAnvilDataButton = (
+    <Modal
+      modalName="addAnvilDataset"
+      title="Add Anvil Dataset"
+      size="small"
+      trigger={<ButtonLink>Request Additional Data</ButtonLink>}
+    >
+      <LoadWorkspaceDataForm params={params} projectGuid={project.projectGuid} />
+    </Modal>
+  )
+
+  const dataButton = !project.isAnalystProject && project.canEdit ? addAnvilDataButton :
```

Comment:
not necessary, but adding parenthesis around `(!project.isAnalystProject && project.canEdit)` would make this more readable